lib/elixir/lib/kernel.ex

```diff
@@ -5288,4 +5288,83 @@ defmodule Kernel do
   defmacro to_char_list(arg) do
     quote(do: Kernel.to_charlist(unquote(arg)))
   end
+
+  @doc """
+  Evaluates the expression and prints additional debugging information.
+
+  The output includes the current file and line, the given expression, the result
+  of evaluating the expression, and time measurement (using `:timer.tc/1`).
+
+  ## Examples
+
+  Imagine you have a `run.exs` file with the following line:
+
+      now = dbg Time.utc_now()
+
+  First, the `Time.utc_now()` expression is evaluated. Then, its result is bound to the
+  `now` variable, and the following is printed:
+
+      run.exs:1: Time.utc_now() #=> ~T[07:03:39.303151] (49µs)
+
+  ## Options
+
+    * `:device` - IO device to write the output to. Defaults to `:stdio`
+
+    * `:inspect` - list of options passed to `Kernel.inspect/2` to pretty-print
+      the result. See `Inspect.Opts` for a full list of options. Defaults to
+      `[pretty: true]`
+
+    * `:label` - print a given label instead of the expression. This is especially
+      useful for pipelines which can expand to very long expressions. Defaults to
+      `Macro.to_string(expr)`
+
+  """
+  @doc since: "1.9.0"
+  defmacro dbg(expr, opts \\ []) do
+    file = Path.relative_to_cwd(__CALLER__.file)
+    line = to_string(__CALLER__.line)
+    default_label = Macro.to_string(expr)
+
+    quote do
+      opts = unquote(opts)
+      device = Keyword.get(opts, :device, :stdio)
+
+      inspect_opts =
+        opts
+        |> Keyword.get(:inspect, [])
+        |> Keyword.put_new(:pretty, true)
```

Comment:
Reading the documentation, I would expect something like `Keyword.get(opts, :inspect, [pretty: true])`. 

I would either update the documentation to express the `:pretty` option is always set unless explicitly specified otherwise, or update the code 🙂